tests/phpunit/ApiTest.php

```diff
@@ -0,0 +1,601 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\SiSenseWriter\Tests;
+
+use GuzzleHttp\Client;
+use Keboola\Component\UserException;
+use Keboola\Csv\CsvReader;
+use Keboola\Csv\CsvWriter;
+use Keboola\SiSenseWriter\Api;
+use Keboola\SiSenseWriter\Config;
+use Keboola\SiSenseWriter\ConfigDefinition;
+use PHPUnit\Framework\TestCase;
+
+class ApiTest extends TestCase
+{
+
+    public function testLogin(): void
+    {
+        $accessToken = $this->getApiConnection()->login();
+        $this->assertNotEmpty($accessToken);
+    }
+
+    public function testInvalidAddress(): void
+    {
+        $this->expectException(UserException::class);
+        $this->expectExceptionMessage('Could not resolve host "http://invalidhost.cz:30845"');
+        $this->getApiConnection(['host' => 'invalidhost.cz'])->login();
+    }
+
+    public function testInvalidLogin(): void
+    {
+        //phpcs:disable Generic.Files.LineLength
+        $expectMessage = 'Client error: `POST http://%s:%s/api/v1/authentication/login` resulted in a `401 Unauthorized` response:
+{"error":{"code":5001,"message":"Invalid domain.","status":401,"httpMessage":"Unauthorized"}}
+';
+        //phpcs:enable
+
+        $this->expectException(UserException::class);
+        $this->expectExceptionMessage(sprintf(
+            $expectMessage,
+            getenv('SISENSE_HOST'),
+            getenv('SISENSE_PORT')
+        ));
+        $this->getApiConnection(['username' => 'invalid.username'])->login();
+    }
+
+    public function testUploadFile(): void
+    {
+        $csvFile = $this->createCsvExampleFile();
+        $api = $this->getApiConnection();
+        $api->login();
+
+        $sisenseFile = $api->uploadFile(
+            $csvFile->getPath() . '/',
+            $csvFile->getFilename()
+        );
+
+        $this->assertStringContainsString('/opt/sisense/storage/datasets/storage/', $sisenseFile);
```

Comment:
Tie subory sa tam po teste aj nejak mazu? Zatial som si to nikde nevsimol a bolo by to asi dobre.

Reply:
jj fajn by to bylo, ale nikde jsem nenašel jak se soubory mažou

Reply:
potom ok.